### PR TITLE
Add a dedicated data privacy page

### PR DESCRIPTION
**Pull request by nrichers**
## Internal Notes for Reviewers

This PR:

- Adds a dedicated data privacy policy page as discussed in Lisbon
- Reworks some existing FAQ and platform overview content that already contained some privacy policy info
- NEW: Add a cookie consent modal and anonymize IP addresses

## Output — data privacy page

Capto_Capture 2023-11-30_08-21-58_am

## Output — cookie consent banner 

image

Preferences: 

image

## External Release Notes

A new data privacy policy page now aggregates all the information that explains how we protect your personal information in one convenient location. [Read more ...](../../guide/data-privacy-policy.qmd)

### CODE REVIEW

File: site/guide/data-privacy-policy.qmd [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+ValidMind's security and privacy controls are designed to align with the stringent requirements of the SOC 2 standard. This compliance means that ValidMind has established and consistently maintains a set of security measures and protocols that meet or exceed the benchmark set by SOC 2. We also regularly review and update these controls to ensure that they stay current with evolving security threats and regulatory requirements. 
+
+## Do you offer additional data privacy options?
+
+ValidMind's platform is a secure, multi-tenant solution hosted on Amazon Web Services (AWS). For organizations that require a stricter trust model and the highest level of security, such as financial services organizations handling highly sensitive data, ValidMind also offers a _Virtual Private ValidMind (VPV)_ option to host our solution in a dedicated single-tenant cloud instance.
```
Comment (mehdi0501) [inline site/guide/data-privacy-policy.qmd:34]:
```suggestion
ValidMind's platform is a secure, multi-tenant solution that can be hosted on Amazon Web Services (AWS), Microsoft Azure Cloud (Azure), or Google Cloud Platform (GCP). For organizations that require a stricter trust model and the highest level of security, such as financial services organizations handling highly sensitive data, ValidMind also offers a _Virtual Private ValidMind (VPV)_ option to host our solution in a dedicated single-tenant cloud instance.
```

File: site/guide/data-privacy-policy.qmd [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The Virtual Private ValidMind option provides all the features and services of other editions of our products, but hosted within a separate environment that is isolated from other ValidMind accounts. VPV accounts do not share resources with non-VPV accounts.
+
+Access to any edition is available through AWS PrivateLink, which provides private connectivity between ValidMind and your on-premises network without exposing your traffic to the public internet.
```
Comment (mehdi0501) [inline site/guide/data-privacy-policy.qmd:38]:
```suggestion
Access to any edition is available through AWS PrivateLink / Azure Privatelink / GCP Private Service Connect, which provides private connectivity between ValidMind and your on-premises network without exposing your traffic to the public internet. 
```

**Review by mehdi0501 — COMMENTED** [review]
Looks good to me, with the suggested edits. Given the legal sensitivity, I would suggest Jonas should review before we publish.

**Comment by nrichers** [comment]
Thank you, @mehdi0501, I have committed your suggestions and thank you for using that feature! Made some minor additional text tweaks via https://github.com/validmind/documentation/pull/148/commits/d58d688ce0b5e27cbc3e8d68b77432606f8a8c71 and will ask @jonasjacobi to review.

**Review by nibalizer — APPROVED** [review]

**Comment by nibalizer** [comment]
Looks good to me

**Pull request merged**